libfreerdp/codec/test/TestFreeRDPRegion.c

```diff
@@ -644,6 +644,9 @@ static int test_norbert_case() {
 			goto out;
 	}
 
+	if (!compareRectangles(region16_extents(&region), &expected_inter_extents, 1) )
```

Comment:
@hardening 
So I copy the extents test here to prove that before the 'intersect' operation, the result is as my expected {2, 0, 1920, 1078}